sesame/backends.py

```diff
@@ -52,6 +53,21 @@ def unsign(self, token):
             data = self.signer.unsign(token, max_age=self.max_age)
         return signing.b64_decode(data.encode())
 
+    def get_hash_value(self, user):
+        """
+        Gets the value to hash - either the password or password and last
+        login for one time tokens
+        """
+        if self.one_time:
+            if user.last_login is None:
+                last_login = ''
+            else:
+                last_login = user.last_login.replace(
+                    microsecond=0, tzinfo=None)
```

Comment:
Is there a downside to just doing `str(user.last_login)`?

I don't think this will make a significant difference in practice. However I'd rather not do anything without a good reason in a security-related library.